tests/e2e/tidbcluster/tls.go

```diff
@@ -310,14 +343,15 @@ spec:
   usages:
     - client auth
   issuerRef:
-    name: {{ .ClusterName }}-tidb-issuer
+    name: {{ .ClusterRef }}-tidb-issuer
     kind: Issuer
     group: cert-manager.io
 `
 
 type tcTmplMeta struct {
 	Namespace   string
 	ClusterName string
+	ClusterRef  string
```

Comment:
Why do we need to add another field? We can pass different `ClusterName` for different TCs, right?